tests/aws-istio-ingress-overlays-cognito_test.go

```diff
@@ -88,15 +88,31 @@ metadata:
   name: istio-ingress
 spec:
   rules:
-    - http:
-        paths:
-          - backend:
-              serviceName: istio-ingressgateway
-              servicePort: 80
-            path: /*
+  - http:
+      paths:
+      - backend:
+          serviceName: istio-ingressgateway
+          servicePort: 80
+        path: /*
+`)
+	th.writeF("/manifests/aws/istio-ingress/base/params.yaml", `
```

Comment:
Could you rerun the make generate? Seems this is duplicate. We should only have one 

```
	th.writeF("/manifests/aws/istio-ingress/base/params.yaml", `
varReference:
- path: metadata/annotations
  kind: Ingress
`)
```

Reply:
Resolved in the latest commit